project3/styles.css

```
@@ -0,0 +1,218 @@
+:root{
+  --color-red:
+}
+*, 
+*::before,
+*::after{
+  box-sizing: border-box;
+}
+
+body{
+  background-color: #163346;
+}
+
+#Shield {
+    height: 239px;
+    width: 240px;
+    color: #000000;
+}
+  h1{
+  font-family: Teko;
+  font-size: 48px;
+  font-weight: 900; 
+  color: #000000;
+}
+h2{
+  font-family: Teko;
+  font-size: 26px;
+  font-weight: 300; 
+  color: #000000;
+ }
+ 
+ legend {
+  font-family: Teko;
+  font-weight: 500;
+  font-size: 40px;
+  border-bottom: solid #163346;
+  margin: 24px 0; 
+}
+
+
+p {
+  color: #cececd;
+  margin: 24px 0; 
+}
+
+
+
+ 
+ fieldset{
+  padding:0;
+  font-family: BenchNine;
```

Missing fallback font info:
```suggestion
  font-family: BenchNine, sans-serif;
```